docs/jumpstart/week-10.md

```diff
@@ -0,0 +1,221 @@
+Next, we have to create a Procfile. This is a text-like file that that acts almost like a configuration file that Heroku uses to set up your java application. Our procfile must look like:
+
+```ignorelang
+web: java $JAVA_OPTS -cp target/classes:target/dependency/* <project main package>
+```
+
+In our case, if you download our catch-up code, our procfile looks like:
+```ignorelang
+web: java $JAVA_OPTS -jar service/target/service-1.0-SNAPSHOT-jar-with-dependencies.jar
+```
+
+What this procfile is doing is specifying that our application will use the HTTP protocol, indicated by the "web" portion and our "java" section specifies it is a Java application. 
+The last portion of the Procfile creates our jar file. A jar file is a file used to run a java application. You can use a jar file to run your java code anywhere. It's easily exported and
+is how Heroku runs our application. 
+
+The next configuration we need a <strong> system.properties </strong> file. This is a file that Heroku uses to define our application's properties such as our Java and Maven version. 
+
+Our system.properties looks like
+```ignorelang
+java.runtime.version=11
+maven.version=3.6.2
+```
+Make sure this file is in our root directory so that Heroku can easily identify this file.
+
+Once we have all of our configurations set up, we can begin deploying our backend. Navigate to our backend directory and run the command:
+
+```shell
+mvn clean install
+```
+
+This will clean our application and build it again.
+Once we verified that our application successfully built, we follow the same steps as deploying our frontend code.
+
+We run the commands:
+
+```shell
+git add .
+```
+which adds all our files to our project's git repository
+
+```shell
+git commit -m
+```
+which commits our added files
+
+We should already be logged into Heroku because we deployed our frontend code but if we're not, then we have to login again using:
+```shell
+heroku login
+```
+
+After logging in we run:
+
+```shell
+heroku create
+```
+which creates a new project on Heroku that represents our backend
+
+```shell
+git push heroku master
+```
+This will push our code to the Heroku server. Again, if we want to deploy a version that is not in our master branch we run the command:
+```shell
+git push heroku <branch_name>:master
+```
+
+After pushing our code to the Heroku server we should be given a URL. This URL is now the URL that our frontend needs to make request to.
+To verify that our backend deployed successfully, we again run the command:
+```shell
+heroku open
+```
+which should open a new browser window with a message at the top saying <strong> Hello Jumpstarters! </strong>
+
+Now that our backend and frontend are deployed, we have to configure our requests URL. We have to navigate back to our frontend project and modify our 
+<strong> .env.production </strong> file and change it from:
+
+~~REACT_APP_API_DOMAIN = "http://localhost:8081"~~
+REACT_APP_API_DOMAIN = "<heroku_backend_url>"
```

Comment:
Good job figuring this out! One minor issue is that this file should not be included in git with the .gitignore. Instead, it should be set in the Heroku secrets